opennms-model/src/test/java/org/opennms/netmgt/model/discovery/IPAddrRangeTest.java

```diff
@@ -72,5 +73,12 @@ public void testIterateSingleton() {
         assertEquals(addr2.toInetAddress(), it.next());
         assertFalse(it.hasNext());
     }
+    
+    public void testGetSizeOfIpAddrRange(){
+    	assertEquals(small.getSizeOfIpAddrRange(),new BigInteger("2"));
```

Comment:
The math here is wrong. small's size should be 3 and singleton's size should be 1.